niceml/utilities/imageloading.py

```diff
@@ -32,6 +33,8 @@ def load_img_uint8(
     """
     file_system: AbstractFileSystem = file_system or LocalFileSystem()
     try:
+        if type(image_path) == LocationConfig:
```

Comment:
Please use `isinstance` instead of `type`
[https://switowski.com/blog/type-vs-isinstance/](https://switowski.com/blog/type-vs-isinstance/)

Reply:
done.